docs/pages/lighting-setups.rst

```diff
@@ -0,0 +1,197 @@
+The LightPositionModel defines the coordinate frame of the light and can be set to any of:
+
+.. code:: python
+
+    habitat_sim.gfx.LightPositionModel.CAMERA
+    habitat_sim.gfx.LightPositionModel.GLOBAL
+    habitat_sim.gfx.LightPositionModel.OBJECT
+
+Each LightSetup is registered in the simulator via a unique key. Two default lighting setups are pre-defined:
+
+
+- **habitat_sim.gfx.DEFAULT_LIGHTING_KEY**
+
+  8 white, GLOBAL lights at scene bounding box corners.
+
+- **habitat_sim.gfx.NO_LIGHT_KEY**
+
+  0 lights (for Flat shaded assets).
+
+
+Additional custom setups can be created and registered via the **simulator.set_light_setup** function:
+
+.. code:: python
+
+    sim.set_light_setup(new_light_setup, "my_custom_lighting_key")
+
+Any existing LightSetup can be queried with **simulator.get_light_setup**:
+
+.. code:: python
+
+    custom_light_setup = sim.set_light_setup(n"my_custom_lighting_key")
+
+
+`Working with Lights Example:`_
+===============================
+
+The example code below demonstrates the usage of default and custom LightSetups and is runnable via:
+
+.. code:: shell-session
+
+    $ python examples/tutorials/lighting_tutorial.py
+
+First, we import modules we will need, define some convenience functions, and initialize the Simulator, Agent, and objects.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [setup]
+    :end-before: # [/setup]
+
+`Scene Lighting`_
+=================
+
+By default, the scene will be shaded with no lights using the **habitat_sim.gfx.NO_LIGHT_KEY** LightSetup.
+This configuration is ideal for scene assets with illumination baked into textures (e.g. building scans such as MP3D).
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [default scene lighting]
+    :end-before: # [scene swap shader]
+
+.. image:: images/lighting-setups-images/0.jpg
+    :width: 45%
+
+To use a custom light setup for the scene, edit the **habitat_sim.SimulatorConfiguration.scene_light_setup**
+option when creating/reconfiguring your Simulator. Note that
+while you can modify the scene's LightSetup dynamically during runtime, you will need to
+reconfigure the simulator to switch the scene's LightSetup key. Also, due to asset loading specifics
+**you must close and re-initialize the Simulator to swap between Flat and Phong shading setups**.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [scene swap shader]
+    :end-before: # [/scene]
+
+
+
+.. |im1| image:: images/lighting-setups-images/1.jpg
+   :width: 100%
+
+.. |im2| image:: images/lighting-setups-images/2.jpg
+   :width: 100%
+
+.. |cp1| replace:: After loading with DEFAULT_LIGHTING_KEY.
+
+.. |cp2| replace:: After swapping to custom "my_scene_lighting".
+
++---------+-----------+
+|  |im1|  +   |im2|   +
+|  |cp1|  +   |cp2|   +
++---------+-----------+
```

Comment:
*(Interesting, it never occured to me to use the || replacements for a table.)*

A table is however problematic for responsive layout -- on small screen sizes it makes the images really tiny and text wrapped badly. The following will place the images next to each other if the screen is wide enough, falling back to placing the images one after another (and limiting them to reasonable width again) if there's not enough space:

```rst
.. container:: m-row

    .. container:: m-col-m-5 m-push-m-1 m-col-s-6

        .. figure:: images/lighting-setups-images/1.jpg
            :figclass: m-flat
            :width: 20em

            ..

            After loading with :ref:`DEFAULT_LIGHTING_KEY <habitat_sim.gfx.DEFAULT_LIGHTING_KEY>`.

    .. container:: m-col-m-5 m-push-m-1 m-col-s-6

        .. figure:: images/lighting-setups-images/2.jpg
            :figclass: m-flat
            :width: 20em

            ..

            After swapping to custom ``my_scene_lighting``.
```

This is how it looks:

![image](https://user-images.githubusercontent.com/344828/78976265-192b8180-7b16-11ea-89a0-c8fef11414da.png)

I get that this is probably not an area of expertise any of you would want to specialize in (mine wasn't either, until I *had to*, haha), but getting familiar with various `m-col-*` things from [Grid Layout](https://mcss.mosra.cz/css/grid/) will empower you to do various tricks to make the tutorials as well as the Habitat website look great and be readable on any screen size (which in my opinion matters a lot and is a big differentiating factor in documentation usability).

Reply:
Thanks! I looked at a couple methods and didn't immediately find one that worked well. This works way better. 😁